lib/report.js

```diff
@@ -1,15 +1,19 @@
+'use strict'
+
+const { readdirSync, readFileSync, statSync } = require('fs')
+const { isAbsolute, resolve, extname } = require('path')
+const { pathToFileURL, fileURLToPath } = require('url')
+const util = require('util')
 const Exclude = require('test-exclude')
 const libCoverage = require('istanbul-lib-coverage')
 const libReport = require('istanbul-lib-report')
 const reports = require('istanbul-reports')
-const { readdirSync, readFileSync, statSync } = require('fs')
-const { isAbsolute, resolve, extname } = require('path')
-const { pathToFileURL, fileURLToPath } = require('url')
-const getSourceMapFromFile = require('./source-map-from-file')
 // TODO: switch back to @c88/v8-coverage once patch is landed.
+const { mergeProcessCovs } = require('@bcoe/v8-coverage')
```

Comment:
Happy to move any `require`s back if you think they affect load time a lot. From my local load benchmark, it seems there's a small penalty ~3-5ms average.